templates/src/error.rs

```diff
@@ -19,6 +19,18 @@ pub enum LoaderError {
         #[snafu(source(from(Vec<fluent_syntax::parser::ParserError>, FluentError::from)))]
         source: FluentError,
     },
+    /// An error was found whilst loading a bundle at runtime.
+    #[snafu(display("Failed to add FTL resources to the bundle"))]
+    FluentBundle {
+        /// The original bundle errors
+        errors: Vec<fluent_bundle::FluentError>
```

Comment:
I think this should also include the path in the error and the error message as I believe the fluent error won't.
```suggestion
        errors: Vec<fluent_bundle::FluentError>
        path: PathBuf,
```

Reply:
I think it makes sense to add the path however with the second block of code that loads the lang specific resources the path information has been lost as it is not stored in the `HashMap` and `FluentResource` does not store the path.

The only easy way I can see to achieve this would be for `crate::fs::read_from_dir` to return a tuple including the path with the FluentResource and then wrap that with `Arc::new`.